config.h.in

```diff
@@ -0,0 +1 @@
+#cmakedefine HAVE_GETENTROPY
```

Comment:
vmemcache currently uses target_compile_definitions for that.
If you really really (I hope not ;) want to use config.h I'd prefer it to be done in a separate PR with conversion of other constants too.